Documentation/Storage-Configuration/Block-Storage-RBD/block-storage.md

```diff
@@ -192,8 +192,47 @@ The OSDs must be located on different nodes, because the [`failureDomain`](../..
 The erasure coded pool must be set as the `dataPool` parameter in
 [`storageclass-ec.yaml`](https://github.com/rook/rook/blob/master/deploy/examples/csi/rbd/storage-class-ec.yaml) It is used for the data of the RBD images.
 
-### Erasure Coded Flex Driver
+### Node Loss
 
-The erasure coded pool must be set as the `dataBlockPool` parameter in
-[`storageclass-ec.yaml`](https://github.com/rook/rook/blob/master/deploy/examples/flex/storage-class-ec.yaml). It is used for
-the data of the RBD images.
+In case, a node is lost where a pod is running with the RBD RWO volume is mounted, the volume cannot automatically be mounted on another node. To recover, confirm the node is down and then add the below taint to the node
+
+!!! Note
+    These instructions are for clusters with Kubernetes version 1.26 or greater. For K8s 1.25 or older, see the [manual steps in the CSI troubleshooting guide](../../Troubleshooting/ceph-csi-common-issues.md#node-loss) to recover from the node loss.
+
+Deploy the csi-addons crds:
+
+```console
+kubectl create -f https://raw.githubusercontent.com/csi-addons/kubernetes-csi-addons/v0.7.0/deploy/controller/crds.yaml
+kubectl create -f https://raw.githubusercontent.com/csi-addons/kubernetes-csi-addons/v0.7.0/deploy/controller/rbac.yaml
+kubectl create -f https://raw.githubusercontent.com/csi-addons/kubernetes-csi-addons/v0.7.0/deploy/controller/setup-controller.yaml
+```
+
+enable the `csi-addons` sidecar in operator yaml configuration
+
+```console
+kubectl patch cm rook-ceph-operator-config -n<namespace> -p $'data:\n "CSI_ENABLE_CSIADDONS": "true"'
+```
+
+```console
+kubectl taint nodes <node-name> node.kubernetes.io/out-of-service=nodeshutdown:NoExecute
+kubectl taint nodes <node-name> node.kubernetes.io/out-of-service=nodeshutdown:NoSchedule
+```
+
+After the taint is added to the node, Rook will blocklist the node to prevent connections to Ceph from the RBD volume on that node. See verify a node is blocklisted:
```

Comment:
```suggestion
After the taint is added to the node, Rook will automatically blocklist the node to prevent connections to Ceph from the RBD volume on that node. To verify a node is blocklisted:
```